tests/improved-sanity-test/vars/common.yml

```diff
@@ -7,10 +7,8 @@ g_file2: 'commit2'
 g_dir2: 'qe2'
 g_hostname: 'myhostname'
 g_pkg: "wget"
-g_rpm_url: 'https://download.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm'
-g_rpm_url_fedora: 'https://kojipkgs.fedoraproject.org/packages/ltrace/0.7.91/22.fc26/x86_64/ltrace-0.7.91-22.fc26.x86_64.rpm'
-g_rpm_name: 'epel-release'
-g_rpm_name_fedora: 'ltrace'
+g_rpm_url: 'https://raw.githubusercontent.com/projectatomic/atomic-host-tests/master/aht-dummy-1.0-1.noarch.rpm'
```

Comment:
That's cool, though we could copy over and install the RPM from our checkout, right? One less thing that relies on external infra. :)

Reply:
We could...but the `rpm_install` role expects a URL to install from.

I could rework the role to handle either a discrete file or a URL...let me investigate if it would be worthwhile.